EventEnabledInsurance/ProjectionClaims/src/main/java/com/ibm/cp4i/demos/eei/projectionclaims/MyHttpHandler.java

```diff
@@ -0,0 +1,187 @@
+                                "table, th, td {" +
+                                    "border: 1px solid black;" +
+                                "}" +
+                                "th, td {" +
+                                    "padding: 5px;" +
+                                "}" +
+                            "</style>" +
+                            "<title>Projection Claim Application</title>" +
+                        "</head>" +
+                        "<body>"
+                    );
+                    if (table == null) {
+                        System.out.println("No records found");
+                        contentBuilder.append("<h2>No records found").append("</h2>");
+                    } else {
+                        System.out.println("Total records found: " + table.size());
+                        contentBuilder.append("<h4>Searched for all table data and found ").append(table.size()).append(" records:").append("</h4>");
+                        contentBuilder.append
+                        (
+                            "<table style=\"width:100%\">" +
+                                "<caption><h4>All claims</h4></caption>" +
+                                "<tr>" +
+                                "<th>QuoteID</th>" +
+                                "<th>Name</th>" +
+                                "<th>Email</th>" +
+                                "<th>Address</th>" +
+                                "<th>US State</th>" +
+                                "<th>License Plate</th>" +
+                                "<th>Claim Status</th>" +
+                            "</tr>"
+                        );
+                        createTableForAllData(table, contentBuilder);
+                    }
+                    contentBuilder.append
+                    (
+                        "</body>" +
+                        "</html>"
+                    );
+                } catch (Throwable exception) {
+                    exception.printStackTrace();
+                }
+                htmlResponse = contentBuilder.toString();
+                httpExchange.sendResponseHeaders(200, htmlResponse.length());
+                outputStream.write(htmlResponse.getBytes());
+                System.out.println("------------------------------------------------");
+            }
+            // get a particular record
+            else if (!requestParamValue.isEmpty()) {
+                System.out.println("Requested for a particular quote id: " + requestParamValue);
+                JSONObject trimmedRow = new JSONObject();
+                byte[] byteResponse;
+                try {
+                    JsonNode row = this.monitor.getRow(requestParamValue);
+                    if (row != null) {
+                        trimmedRow.put("quoteid", row.get("quoteid").toString().replace("\"", ""));
+                        trimmedRow.put("name", row.get("name").toString().replace("\"", ""));
```

Comment:
Why is this removing double quotes? I guess the original data could have double quotes? Or is `toString` returning leading/trailing quotes? If the original data has double quotes in it then we want them to be preserved and returned in the json.

Reply:
If the replace is removed, the JSON returned looks like:
```
{
    "usstate": "\"VA\"",
    "address": "\"790 Arrowhead Court, Portsmouth\"",
    "claimstatus": "2",
    "name": "\"Ronny Doyle\"",
    "licenseplate": "\"WMC-9628\"",
    "quoteid": "183",
    "email": "\"RonnyDoyle@mail.com\""
}
```

Reply:
Done